### PR TITLE
CAMS-124 Refactoring ops structure to improve screaming architecture

### PR DESCRIPTION
**Pull request by magile32**
# Purpose

Contents of `\ops\` folder has grown. Refactoring for screaming architecture improvements.  

# Major Changes

- Updates to ops folder structure
- Update existing scripts and bicep that was effected by path changes.

# Testing/Validation

- Successful branch deployment
- Manual execution of git setup script 

# Notes

NEEDS pipeline.yaml updates on the USTP environment after merging to main.

### CODE REVIEW

File: docs/ops.md [hunk 1]
```diff
@@ -4,26 +4,42 @@ Contents here contains supporting operational scripts and tools outside the scop
...
+Shell scripts executed in the CI/CD pipeline.
+
+### /scripts/utility
+
+Contain standalone helper scripts executed on local dev machines.
+
+## /git-setup
+
+Execute set-up-git-hooks.sh from root of repository to configure pre-commit hooks and leverage a commit template.
+
```
Comment (jamesobrooks) [inline docs/ops.md:42]: This is already captured in contributing.md. I don't think we will want to maintain it in two places.

Comment (magile32) [reply docs/ops.md:42]: @jamesobrooks What change do you recommend?

Whats in the ops.md describes the contents of the high level folders under ops. Contributing seems to have a different purpose.

**Review by amorrow-flexion — APPROVED** [review]

File: docs/ops.md (hunk 1 above)
Comment (magile32) [reply docs/ops.md]: Updated. Moved some of the ops and deployment content to new operations folder.

File: docs/operations/scripts.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<!-- docs/operations/_sidebar.md -->
+
+* [< Back](/)
+* [Deployment](/operations/deployment.md)
+* [Scripts](/architecture/scripts.md)
```
Comment (jamesobrooks) [inline docs/operations/scripts.md:5]: This content looks like a `_sidebar` file's contents.

**Review by jamesobrooks — APPROVED** [review]

**Pull request merged**